frontend/src/metabase/nav/containers/MainNavbar/BookmarkList/BookmarkList.styled.tsx

```diff
@@ -25,5 +44,25 @@ export const SidebarBookmarkItem = styled(SidebarLink)`
       props.isSelected ? color("text-white") : color("brand")};
     cursor: pointer;
     margin-top: 3px;
+
+    > svg:focus {
+      outline: none;
+    }
   }
+
+  ${props =>
+    props.isSorting &&
+    css`
+      &:hover {
+        background: white;
+
+        svg {
```

Comment:
You can use 

```
${DragIcon} {
  color: ...
}
```;

to style the icon instead of relying on html tags

Reply:
Thank you @ranquild.

I've just tried this from your feedback.

Here we're controlling the style both for the DragIcon svg and the icon next to the name of the item.

I don't know if it's advantageous to refer to the components in this case, as it can get more verbose.
Since we are indeed styling the svg's, I'd believe that this way is terse and understandable.

Wdyt?

Reply:
I don't have good arguments to justify using component references here. I just thought that it's not obvious that components could be used like that, and wanted to share this info.